ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Nagios_Plugins/tasks/nagios_CentOS.yml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
What's the --- for?

Reply:
That's a YAML document separator: https://yaml.org/spec/1.0/#id2561718